.github/scripts/check_migrations.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+import sys
+
+
+def get_diff():
+    subprocess.run(
+        ["git", "fetch", "origin"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL
+    )
+    output = [
+        f"./{file_path}"
+        for file_path in subprocess.check_output(
+            ["git", "diff", "--name-only", "origin/develop"]
+        )
+        .decode()
+        .splitlines()
+    ]
+    if not output:
+        output = [
+            f"./{file_path}"
+            for file_path in subprocess.check_output(
+                ["git", "diff", "--name-only", "origin/main"]
```

Comment:
same as above `origin/main` what will happen if the operations are moved to new branch names?